x-pack/plugin/transform/src/test/java/org/elasticsearch/xpack/transform/transforms/TransformIndexerStateTests.java

```diff
@@ -134,7 +132,7 @@
                 context
             );
             this.threadPool = threadPool;
-            this.executorName = executorName;
+            this.executorName = ThreadPool.Names.GENERIC;
```

Comment:
not really important since this is just a test, but do we need to keep this `executorName` instance variable if it is static?